runtime/doc/lua.txt

```diff
@@ -2211,6 +2211,24 @@ input({opts}, {on_confirm})                                   *vim.ui.input()*
                       typed (it might be an empty string if nothing was
                       entered), or `nil` if the user aborted the dialog.
 
+os_open({path})                                             *vim.ui.os_open()*
```

Comment:
After more thought, I'm pretty certain that `vim.ui.open` is good. There could be `vim.fs.open` also, but they're in separate namespaces and the purpose is clear: "fs" is for filesystem, "ui" is where platform-specific providers live.

Reply:
```suggestion
open({path})                                             *vim.ui.open()*
```